pallets/parachain-staking/src/auto_compound.rs

```diff
@@ -63,6 +63,11 @@ where
 		Self(sorted_delegations)
 	}
 
+	/// Retrieves the length of an instance of [AutoCompoundingDelegations] storage.
```

Comment:
I would be clear that this returns the number of elements and not size in bytes, either through additional comments or the fn name itself. The upstream documentation isn't very clear about this point, either. I think this is the "source of truth" on that matter:

https://github.com/paritytech/parity-scale-codec/blob/e2c4ff3f12b836b1ffcc05f57192f4aa4b471c58/src/codec.rs#L283

Reply:
You are right. Done.